.github/workflows/python_installation.yml

```diff
@@ -28,10 +32,8 @@ jobs:
         conda install -n base conda-libmamba-solver
         conda config --set solver libmamba
```

Comment:
From 23.10 this should be the default behaviour: https://conda.org/blog/2023-11-06-conda-23-10-0-release/

I think we're just getting the latest version so this should no longer be necessary (same in other workflows, but maybe just test to be sure)